src/backgrounds.jl

```diff
@@ -0,0 +1,164 @@
+        # Bottom to top for vertical grid lines
+        for x_point = min_width:line_gap:max_width
+            start_point = Point(x_point, max_height)
+            finish_point = Point(x_point, min_height)
+            end_point = start_point + step * (finish_point - start_point)
+            line(start_point, end_point, :stroke)
+        end
+    else
+        # Top to bottom motion for vertical grid lines
+        for x_point = min_width:line_gap:max_width
+            start_point = Point(x_point, min_height)
+            finish_point = Point(x_point, max_height)
+            end_point = start_point + step * (finish_point - start_point)
+            line(start_point, end_point, :stroke)
+        end
+    end
+
+    if direction[2] == 'R'
+        # Left to right motion for horizontal grid lines
+        for y_point = min_height:line_gap:max_height
+            start_point = Point(min_width, y_point)
+            finish_point = Point(max_width, y_point)
+            end_point = start_point + step * (finish_point - start_point)
+            line(start_point, end_point, :stroke)
+        end
+    else
+        # Right to left motion for horizontal grid lines
+        for y_point = min_height:line_gap:max_height
+            start_point = Point(max_width, y_point)
+            finish_point = Point(min_width, y_point)
+            end_point = start_point + step * (finish_point - start_point)
+            line(start_point, end_point, :stroke)
+        end
+    end
+
+end
+
+"""
+
+`zero_lines(video::Video, action::Action, frame::Int; direction::AbstractString = "TR", line_thickness = 10)`
+
+Draws zero lines on the given frame of a Video. A closure for the [`_zero_lines`](@ref) internal function.
+
+# Arguments
+- `direction::AbstractString`: Direction for how vertical and horizontal axes are drawn. Default: `"TR"` Available Orientations:
+  - `"TR"` - Vertical axis drawn towards the **T**op and horizontal axis drawn to the **R**ight of the frame.
+  - `"TL"` - Vertical axis drawn towards the **T**op and horizontal axis drawn to the **L**eft of the frame.
+  - `"BR"` - Vertical axis drawn towards the **B**ottom and horizontal axis drawn to the **R**ight of the frame.
+  - `"BL"` - Vertical axis drawn towards the **B**ottom and horizontal axis drawn to the **L**eft of the frame.
+- `line_thickness`: Defines the thickness of the zero lines. Default: `10`
+
+# Example
+This example will produce an animation with the vertical axis being drawn towards the top and the horizontal axis being drawn towards the left. One will need to define their own path for `tempdirectory` and `pathname`.
+
+```
+...
+ Action(1:100, :line, zero_lines(direction = "TL", line_thickness = 10)),
+...
+```
+
+"""
+function zero_lines(; direction::AbstractString = "TR", line_thickness = 10)
+    return (video, action, frame) -> _zero_lines(
+        video,
+        action,
+        frame;
+        direction = direction,
+        line_thickness = line_thickness,
+    )
```

Comment:
this looks totally different than the one in draw_lines. I like this style :wink: